qiskit/compiler/assemble.py

```diff
@@ -248,6 +249,12 @@ def _parse_pulse_args(backend, qubit_lo_freq, meas_lo_freq, qubit_lo_range,
                 meas_freq_est=backend_config_defaults.get('meas_freq_est')
             )
 
+        if meas_level not in backend_config.meas_levels:
+            raise SchemaValidationError(
+                f'meas_level = {meas_level} not supported for backend {backend_config.backend_name}, ' +
+                f'only {backend_config.meas_levels} is supported'
```

Comment:
Looks like this line is too long for the linter (limit is 100 characters). Also, f-strings are only available in python 3.6+, but qiskit still supports python3.5 (though deprecated).